includes/class-wcmp-frontend-settings.php

```diff
@@ -185,36 +207,46 @@ public function is_pickup_enabled() {
         /**
          * @return mixed
          */
-        public function at_home_delivery_titel() {
-            return $this->settings['at_home_delivery_titel'];
+        public function at_home_delivery_title() {
+            if (isset($this->settings['signed_title'])) {
+                return $this->settings['signed_title'];
+            }
         }
 
         /**
          * @return mixed
          */
-        public function be_at_home_delivery_titel() {
-            if (isset($this->settings['be_at_home_delivery_titel'])) {
-                return $this->settings['be_at_home_delivery_titel'];
+        public function be_at_home_delivery_title() {
```

Comment:
Ik zie het nu pas, maar be_at_home is dat een bezorgoptie? Wat is dat? Hebben we daar niet een andere titel voor?

Reply:
Done!